roles/openssl_cert_gen/templates/xdmod.conf

```diff
@@ -0,0 +1,50 @@
+<IfModule !mod_ssl.c>
+    LoadModule ssl_module modules/mod_ssl.so
+</IfModule>
+
+Listen 443
+
+<VirtualHost *:80>
+    ServerName {{ ServerName }}
+    Redirect permanent / https://{{ ServerName }}/
+</VirtualHost>
+
+<IfModule mod_ssl.c>
+    <VirtualHost *:443>
+        ServerName {{ ServerName }}
+        DocumentRoot /usr/share/xdmod/html
+
+        SSLEngine on
+        SSLCertificateFile /etc/ssl/certs/ansible_ssl.crt 
+        SSLCertificateKeyFile /etc/ssl/certs/ansible_ssl.pem
+
+        <Directory /usr/share/xdmod/html>
+            Options FollowSymLinks
+            AllowOverride All
+            DirectoryIndex index.php index.html
+
+            <IfModule mod_authz_core.c>
+                Require all granted
+            </IfModule>
+        </Directory>
+
+        <Directory /usr/share/xdmod/html/rest>
+            RewriteEngine On
+            RewriteRule (.*) index.php [L]
+        </Directory>
+
+        SetEnv SIMPLESAMLPHP_CONFIG_DIR /etc/xdmod/simplesamlphp/config
+        Alias /simplesaml /usr/share/xdmod/vendor/simplesamlphp/simplesamlphp/www
```

Comment:
So it looks like we're still using simplesaml that comes with xdmod, rather than using the simplesaml that we install

Reply:
This follows on the next line as well

Reply:
done